### ISSUE TITLE
PhysBone contacts are not scaled right and missing collision tags and parameters

### ISSUE THREAD

**Issue by Provini** [killfrenzy96/KillFrenzyAvatarText#4]
I am using the latest SDK and release of this software. Example shown below.
On an added note, the example has the same issue. Maybe the SDK changed something?

![image](https://user-images.githubusercontent.com/71570801/171605865-b4fee0e7-3b32-422e-b0ea-3f5f1001cd4f.png)
![image](https://user-images.githubusercontent.com/71570801/171606239-4285a155-8cf2-4440-a2a1-46270333e489.png)

**Comment by killfrenzy96** [comment]
Hi, I just updated the SDK and the contacts seem to work correctly. I do have future updates in mind. Do you have any other scripts that have compilation errors or perhaps interfering?

**Comment by Provini** [comment]
No errors it appears but I'll try removing some stuff and will return with updates

**Comment by Provini** [comment]
Alright so this is weird.

So before I submitted this issue report, I did import the latest SDK at the time along with this script. I didn't realize there were persistent errors at the time and I resolved those.

After that, I installed it to the avatar and noticed touch functionality wasn't functioning in-game and I saw the image above.

Deleting and reimporting the script didn't fix anything, however, right-clicking the KillFrenzy folder and clicking re-import did fix it. Not sure what the difference in that is but consider this resolved.
